app/jobs/tracks/daily_generation_job.rb

```
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+# Daily job to handle bulk track processing for users with recent activity
+# This serves as a backup to incremental processing and handles any missed tracks
+class Tracks::DailyGenerationJob < ApplicationJob
+  queue_as :tracks
+
+  def perform
+    Rails.logger.info "Starting daily track generation for users with recent activity"
```

RuboCop (1.76.1)

[convention] Prefer single-quoted strings when you don't need string interpolation or special symbols.

(Style/StringLiterals)